.storybook/project.json

```diff
@@ -18,7 +18,7 @@
 		"build": {
 			"configurations": {
 				"ci": {
-					"commands": ["cross-env NODE_OPTIONS=\"--no-warnings\" storybook build --config-dir ."]
+					"commands": ["cross-env NODE_OPTIONS=\"--no-warnings\" storybook build --test --config-dir ."]
```

Comment:
Should this `--test` change be here? Was that for local testing?

Reply:
It was recommended in the Storybook blog to optimize the performance of the build. It has to do with how files are broken up.

Reply:
https://storybook.js.org/docs/api/main-config/main-config-build#test
It looks like by default this disables a lot of things for the purpose of testing performance. Would we want all of those things disabled? I see it removes the blocks package used for docs blocks, mdx documentation, and autodocs?

Reply:
Right but it's just being added to the CI command which is used for VRT. It depends if we start using chromatic's build as a full example of the docs site. Currently, we used a netlify build for that which builds using a different command.